beginner_source/onnx/intro_onnx.py

```diff
@@ -0,0 +1,59 @@
+"""
+**Introduction to ONNX** ||
```

Comment:
Maybe consolidate all this in this PR: https://github.com/pytorch/tutorials/pull/2550?

Reply:
That depends on when ONNX Runtime 1.16 will be released. This PR doesn't pass CI without it while #2550 does not need it.

I am monitoring the ort 1.16 release and will proceed as needed

Reply:
@svekars ORT 1.16 is out and the CI issue seems to be gone

There is another unrelated one, though:

```bash
wget -nv 'https://docs.google.com/uc?export=download&id=1HJV2nUHJqclXQ8flKvcWmjZ-OU5DGatl' -O _data/lenet_mnist_model.pth
https://docs.google.com/uc?export=download&id=1HJV2nUHJqclXQ8flKvcWmjZ-OU5DGatl:
2023-09-20 19:46:41 ERROR 429: Too Many Requests.
```